doc/tuning.rst

```diff
@@ -96,6 +96,12 @@ You might be able to get a little more performance by using
 selections) rather than using :meth:`~.DataSet.select` with
 ``scans='track'``.
 
+When using MVF v4 one can also pass an `index` parameter to :meth:`katdal.open`
```

Comment:
I'm not sure if `index` is the best name, but it emphasises that it takes an index expression. Another candidate would be `select`, since it provides functionality similar to `DataSet.select`, but with different syntax.

Reply:
Yes, I share your sentiments -- the word `index` probably elicits no response in most users 😄  And `select` suggests that it will have the same syntax. I also considered `preselect`.

One option would be to go the actual `select` route, but only support `channels=slice()` and `dumps=slice()`. Then there is scope to expand this in future. And it also provides a simple way to handle older formats: simply slap it on top of the initial selection... All formats end their initialisation with a `select()` step (usually to set the spectral window, subarray and antenna list).

Reply:
Oh wait, the dataset dump/channel origin shifts so it is not a pure `select`. Mmm, then the `select` format might also not be that useful.

Reply:
Yes, was just about to point that out.

I am starting to like `select=dict(channels=slice(...), dumps=slice(...))`. It's familiar, it's explicit about which axis is which, and it's theoretically extensible. I was worried that it would confuse users into thinking it was equivalent to a `select` after opening, but since you made that mistake with the current name, maybe it's doomed to be confusing no matter what we call it.

Reply:
I've pushed a change to use `select`. Let me know what you think. At the moment I make the switch over to `index` between Datasource and ChunkStoreVisFlagsWeights, but I'm wondering if it would make more sense to move it to the interface between ChunkStoreVisFlagsWeights and ChunkStore (because ChunkStoreVisFlagsWeights is for specific sorts of data where it makes sense to talk about the "channels" axis, whereas ChunkStore is agnostic).

Also I'd still like your thoughts on how to handle pre-v4.

Reply:
You could still consider `preselect`. For me it suggests `select` notation but the `pre` part warns that it is not exactly the same.

I always considered `select` a top-level feature at the `VisDataV4` level, and would probably have switched to `index` in `TelstateDataSource` already :-D But I like the current scheme and I'm comfortable with it moving down to `VisFlagsWeights` too.